.github/workflows/codecov.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: ./gradlew clean msal:assembleLocal
 
       - name: Run Coverage
-        run: ./gradlew msal:localDebugMsalUnitTestCoverageReport -ProbolectricSdkVersion=$ROBOELECTRIC_SDK_VERSION
+        run: ./gradlew msal:localDebugMsalUnitTestCoverageReport -ProbolectricSdkVersion=$ROBOELECTRIC_SDK_VERSION -PcodeCoverageEnabled=true
```

Comment:
I think we need `-Plabtest` here. We have lots of E2E tests that use the Lab Api. Since these are E2E, they are going to provide lots of coverage. We need to know about these coverage details in the coverage report.

Reply:
> 

The main issue that I encountered is setting up the lab api automation for installing certificates and fetching credentials since this is being ran from a Github action. I would appreciate if you could float me any ideas you have around achieving this

Reply:
Codecov does support Azure Pipelines as CI Provider: https://docs.codecov.com/docs/supported-ci-providers

Reply:
Kindly note that this pipeline is already in place now - https://dev.azure.com/IdentityDivision/Engineering/_build?definitionId=1522&_a=summary

Reply:
Does the report include coverage for code in common? Doesn't look like it. I'm asking because E2E tests in MSAL provide a lot of coverage for the code in common lib

Reply:
Common has its own code coverage pipeline which targets tests within it so is there still need to target the same in MSAL?

Reply:
We need to see what sort of coverage for common code are we getting for MSAL tests...so to answer your question, yes we need to do it.